assets/css/style.css

```diff
@@ -0,0 +1,597 @@
+* {
```

Comment:
One Linter check is not passing, Kindly fix it 👍 😄 

![image](https://user-images.githubusercontent.com/87448628/159117432-f41aa2dd-4bf9-4a94-95ed-1fda6524d8ed.png)

Reply:
Okay